docs/line.rst

```diff
@@ -0,0 +1,112 @@
+.. include:: common.txt
+
+==================
+:mod:`pygame.Line`
+==================
+
+.. currentmodule:: pygame
+
+.. class:: Line
+
+    | :sl:`pygame object for storing line coordinates`
+    | :sg:`Line(x1, y1, x2, y2) -> Line`
+    | :sg:`Line((x1, y1), (x2, y2)) -> Line`
+    | :sg:`Line(object) -> Line`
+
+    Pygame uses Line objects to represent lines using coordinates and provide
+    useful methods to work with lines. A line can be created from a combination of
+    x1, y1, x2 and y2 values. Lines can also be created from python objects that
+    are already a Line or have an attribute named "line".
+
+    Any pygame function that requires a Line argument also accepts any of these
+    values to construct a Line. This makes it easier to create Lines on the fly
+    as arguments to functions.
+
+    The Line object is also iterable:
+
+    ::
+
+      line = Line(0, 1, 2, 3)
+      x1, y1, x2, y2 = line
+
+
+    .. method:: copy
+
+        | :sl:`copy the line`
+        | :sg:`copy() -> line`
+
+        Returns a new Line having the same coordinates as the original
+
+        .. ## Line.copy ##
+
+    .. method:: raycast
+
+        | :sl:`cast a ray against a list of shapes/colliders`
+        | :sg:`raycast(List[Union[Line, Rect, Circle]]) -> Union[Tuple[float, float], None]`
+
+        Performs a raycast against a list of supported shapes, and returns whether the line
+        intersects with each of the shapes or not.
```

Comment:
This kinda makes it soud like this function returns a bool. Could be enough to say `...and returns the closest intersection found, None if there aren't any.`
And right below this you could rephrase to 
`The intersection point is a tuple containing the x and y coordinate of the closest intersection point.`
This way it's simpler, more compact and easier to read.

Another thing to add could be a list of the shapes you can raycast against, that are `Line,` `Circle`, `Rect` and `Polygon`. Also specify that the behaviour of raycast() is always the same, no matter the shape used. The sequence can either be a homogeneous sequence or an heterogeneus one, behaviour will always be the same.